### DISCUSSION TITLE
Vulkan: FogVolume - Add an option to the Fogvolume node Inspector to set the density to a large negative value in order to remove fog in an area, (ex: inside a house, etc...)

### DISCUSSION THREAD

**Discussion by thoced** [godotengine/godot-proposals#5493] [3D]
Hello,

Currently, to allow a FogVolume node to play the role of fog suppression (ex: interior of a house which should not be impacted by volumetric fog), you must manually set the value of the density (FogMaterial) to a large negative value. (_value greater than the volumetric Fog density of the environment)_

I propose adding a button in the inspector to set a large negative value to the density.

This button could be called _"interior"_ or _"negative fog"_ or another name

![image](https://user-images.githubusercontent.com/10273261/192161095-9c0162f1-7b3c-4347-aa86-564c9e71cb23.png)

What do you think about this ?

Throced

**Comment by Calinou** [comment]
Done in https://github.com/godotengine/godot/pull/66444.

> I propose adding a button in the inspector to set a large negative value to the density.

No need to add a button – tweaking the property hint range will do the job :slightly_smiling_face:

**Reply by thoced** [reply]
Ok, changing the density range will indeed do the trick, I was thinking of the button to make the option more understandable for users.
